graylog-storage-opensearch2/src/test/java/org/graylog/storage/opensearch2/views/export/OpenSearchExportBackendIT.java

```diff
@@ -38,6 +38,7 @@
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
```

Comment:
The annotation is from JUnit 5; we are using JUnit 4 in this class. Does that work?

Reply:
No, my fault. I am just so used to JUnit 5 these days. Fixed it, thanks!